addons/website/static/src/builder/plugins/options/portal_option.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="website.PortalOption">
+        <BuilderRow label.translate="Background Color">
+            <BuilderColorPicker
+                defaultColor="''"
+                action="'customizeWebsiteColor'"
+                actionParam="{
+                    mainParam: 'portal-card-custom',
+                    gradientColor: 'portal-gradient',
+                    combinationColor: 'portal-card',
+                }"/>
+        </BuilderRow>
+        <BorderConfigurator label.translate="Border" withBSClass="false" action="'setStylePortalCard'"/>
+    </t>
+    <t t-name="website.PortalCardVisibilityOption">
+            <BuilderRow t-if="this.entries.length" label.translate="Cards Design">
+                    Global options for all inner cards.
+            </BuilderRow>
+        <t t-foreach="this.entries" t-as="entry" t-key="entry.id">
+            <BuilderRow label="entry.name" level="1">
+                <BuilderCheckbox action="'cardVisibilityOption'" actionValue="entry.id" preview="false"/>
+            </BuilderRow>
```

Comment:
You don t need to use `preview=false`.  You builder action is already not previewable